tests/test_create_agent.py

```
@@ -0,0 +1,697 @@
+            # Corrupt the launcher to cause failure during launcher update
+            launcher_file = tmp_path / "agents" / "launch"
+            launcher_file.write_text("#!/bin/bash\n# corrupted")
+
+            result = run_script(
+                ["cleanup-test-agent", "Testing cleanup on failure"],
+                cwd=tmp_path,
+            )
+
+            if result.returncode != 0:
+                # Agent file should not exist (was cleaned up)
+                agent_file = tmp_path / ".claude" / "agents" / "cleanup-test-agent.md"
```

Ruff (0.14.14)

[error] Local variable `agent_file` is assigned to but never used

Remove assignment to unused variable `agent_file`

(F841)